docs/releases.md

```diff
@@ -1,16 +1,18 @@
 ## Checklist
 
 Helix releases are versioned in the Calendar Versioning scheme:
-`YY.0M(.MICRO)`, for example, `22.05` for May of 2022. In these instructions
+`YY.MM.MICRO`, for example, `22.05.0` for May of 2022. In these instructions
```

Comment:
I personally don't like this notational change and would prefer we keep the original format.

Note that `MM` should also be changed back to `0M` regardless 

MM - Short month - 1, 2 ... 11, 12
0M - Zero-padded month - 01, 02 ... 11, 12

https://calver.org/#scheme

Reply:
>Note that MM should also be changed back to 0M regardless

Ah sorry, I was not familiar with CalVer notation until now. I had assumed it was a typo.

>I personally don't like this notational change and would prefer we keep the original format.

So just to clarify, are you talking about just the 0M thing, or having a patch version at all? Because Cargo throws an error without one.